### DISCUSSION TITLE
'airflow' became '***' in log

### DISCUSSION THREAD

**Discussion by gitbebop** [apache/airflow#34299] [Q&A]
we use Airflow v2.4.3, and have a strange problem, 'airflow' became '***' in log

Dag is very simple, just print airflow

```python
def test():
    print('airflow airflow airflow bairflowf')


test_op = PythonOperator(task_id='test',
                                  python_callable=test,
                                  retries=0,
                                  dag=dag)

test_op
```

check log from UI
```
[2023-09-12, 16:46:25 CST] {logging_mixin.py:137} INFO - *** *** *** b***f
[2023-09-12, 16:46:25 CST] {python.py:177} INFO - Done. Returned value was: None
```

check log file in server
```
[2023-09-12T16:46:25.921+0800] {logging_mixin.py:137} INFO - *** *** *** b***f
[2023-09-12T16:46:25.921+0800] {python.py:177} INFO - Done. Returned value was: None
```

Why 'airflow' became '***' in log?

**Comment by Taragolis** [comment]
> Why 'airflow' became '***' in log?

[Masking sensitive data](https://airflow.apache.org/docs/apache-airflow/stable/security/secrets/mask-sensitive-values.html#masking-sensitive-data) will help you for figure out why it replaced  by `***` in logs 😉

**Reply by potiuk** [reply]
HInt. Check the passwords you are using :)

**Reply by gitbebop** [reply]
thanks a lot.
